kedro_kubeflow/generators/pod_per_node_pipeline_generator.py

```diff
@@ -3,6 +3,7 @@
 
 import kubernetes.client as k8s
 from kedro.framework.context import KedroContext
+from kedro.framework.project import pipelines
```

Comment:
Please leave the import where it was  - the `pipelines` object has lazy initialisation procedure and I'm not sure whether it will work when called outside of the initialised Kedro project.

Reply:
This is weird. Lazy initialization would mean that it's done when it's used - so importing it earlier should not matter?